build/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:bullseye@sha256:71cb300d5448af821aedfe63afd55ba05f45a6a79f00dcd131b96b780bb99fe4
+
+ENV CORE_RS_ALBATROSS_COMMIT_HASH=93066288a6bfccc43498ae5596cdf2189b244785
```

Comment:
Can we use the latest tag/hash from the repository instead?

Reply:
Please refer to this comment: https://github.com/nimiq/core-rs-albatross/pull/2164#discussion_r1471619960